packages/next-dashboard/sass/components/_right-sidebar.scss

```diff
@@ -0,0 +1,60 @@
+.right-sidebar {
+  $right-sidebar-width: 300px;
```

Comment:
I'm not 100% if this should be here or if this should be declared in the variables file since I think we have all other declarations in there.

Personally It can be either here or on the variables file, just wanted to make a comment about it. 🙂